[BUG]: Can't launch app

### Checked for duplicates?

- [X] This issue is not a duplicate

### Does it also happen in the desktop version?

- [X] This bug does not occur in the latest version of Anki

### What are the steps to reproduce this bug?

I updated to 2.19 alpha 5 and I can't launch the app now. It crashes as soon as I click the app icon in the launcher.

### Expected behaviour

App works

### Actual behaviour

App doesn't open

### Debug info

```text
Can't send because the app doesn't launch. But, I am sure that you can find the related crash report based on the time (there won't be many reports from this release yet).
```


### (Optional) Anything else you want to share?

_No response_

### Research

- [X] I am reporting a bug specific to AnkiDroid (Android app)
- [X] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [X] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

```
ACRA caught a NoClassDefFoundError for com.ichi2.anki
        java.lang.NoClassDefFoundError: Failed resolution of: Lcom/ichi2/anki/common/utils/android/TestUtilsKt;
        	at com.ichi2.compat.CompatV24.normalize(SourceFile:6)
        	at com.ichi2.anki.AndroidTtsVoice$Companion.toAnkiTwoLetterCode(SourceFile:12)
        	at com.ichi2.anki.AndroidTtsVoice.<init>(SourceFile:47)
        	at com.ichi2.anki.TtsVoicesKt.toTtsVoice(SourceFile:13)
        	at com.ichi2.anki.TtsVoices.loadTtsVoicesData(SourceFile:144)
        	at com.ichi2.anki.TtsVoices.access$loadTtsVoicesData(SourceFile:1)
        	at com.ichi2.anki.TtsVoices$loadTtsVoicesData$1.invokeSuspend(Unknown Source:11)
        	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(SourceFile:12)
        	at kotlinx.coroutines.DispatchedTask.run(SourceFile:129)
        	at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(SourceFile:4)
        	at kotlinx.coroutines.scheduling.TaskImpl.run(SourceFile:3)
        	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(SourceFile:1)
        	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(SourceFile:15)
        	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(SourceFile:29)
        	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(SourceFile:1)
        	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@10da358, Dispatchers.IO]
        Caused by: java.lang.ClassNotFoundException: com.ichi2.anki.common.utils.android.TestUtilsKt
        	... 15 more
```

Could be the different compileOptions in the common module?

Same issue here


```
FATAL EXCEPTION: main
Process: com.ichi2.anki, PID: 20352
java.lang.NoClassDefFoundError: Failed resolution of: Lcom/ichi2/anki/common/utils/android/TestUtilsKt;
	at com.ichi2.utils.Permissions.canManageExternalStorage(SourceFile:6)
	at com.ichi2.anki.InitialActivityKt.selectAnkiDroidFolder(SourceFile:8)
	at i.b.a(SourceFile:6)
	at com.ichi2.anki.DeckPicker.collectionPermissionScreenWasOpened(SourceFile:1)
	at com.ichi2.anki.DeckPicker.handleStartup(SourceFile:1)
	at com.ichi2.anki.DeckPicker.onCreate(SourceFile:128)
	at android.app.Activity.performCreate(Activity.java:8397)
	at android.app.Activity.performCreate(Activity.java:8370)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1403)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3837)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4006)
	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:111)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2462)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:240)
	at android.os.Looper.loop(Looper.java:351)
	at android.app.ActivityThread.main(ActivityThread.java:8377)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:584)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1013)
Caused by: java.lang.ClassNotFoundException: com.ichi2.anki.common.utils.android.TestUtilsKt
	... 22 more
```

--- a/common/build.gradle.kts
+++ b/common/build.gradle.kts
@@ -27,11 +27,11 @@ android {
         }
     }
     compileOptions {
-        sourceCompatibility = JavaVersion.VERSION_1_8
-        targetCompatibility = JavaVersion.VERSION_1_8
+        sourceCompatibility = JavaVersion.VERSION_11
+        targetCompatibility = JavaVersion.VERSION_11
     }
     kotlinOptions {
-        jvmTarget = "1.8"
+        jvmTarget = "11"
     }
 }
 

--- a/common/proguard-rules.pro
+++ b/common/proguard-rules.pro
@@ -19,3 +19,17 @@
 # If you keep the line number information, uncomment this to
 # hide the original source file name.
 #-renamesourcefileattribute SourceFile
+
+# FIXME remove this entire Android 4.2 workaround from 12/3/15 timrae commit for 2.15.x+
+# Samsung Android 4.2 bug workaround
+# https://code.google.com/p/android/issues/detail?id=78377
+-keepattributes **
+-keep class !android.support.v7.view.menu.**,!android.support.design.internal.NavigationMenu,!android.support.design.internal.NavigationMenuPresenter,!android.support.design.internal.NavigationSubMenu,** {*;}
+#5806 - Class: ActionBarOverflow
+-keep public class android.support.v7.internal.view.menu.** { *; }
+-keep public class androidx.appcompat.view.menu.** { *; }
+-dontpreverify
+-dontoptimize
+-dontshrink
+-dontwarn **
+-dontnote **

